src/serviceFacades/tools.js

```diff
@@ -118,6 +118,25 @@ function toolRequest(request) {
     });
 }
 
+function deleteTool({ id }) {
+    return callApi({
+        endpoint: `/api/tools/${id}`,
+        method: "DELETE",
+    });
+}
+
+function deleteTools({ ids: toolIds }) {
+    return (
+        toolIds &&
+        toolIds.length > 0 &&
+        new Promise((resolve, reject) =>
+            Promise.all(toolIds.map((id) => deleteTool({ id })))
+                .then((values) => resolve(values))
+                .catch((error) => reject(error))
+        )
```

Comment:
It turns out the `new Promise((resolve, reject)` wrapper isn't needed:
```suggestion
        Promise.all(toolIds.map((id) => deleteTool({ id })))
```